brave-jersey2/src/test/java/com/github/kristofa/brave/jersey2/ITBraveTracingFeature_Server.java

```diff
@@ -0,0 +1,75 @@
+package com.github.kristofa.brave.jersey2;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.http.ITHttpServer;
+import com.github.kristofa.brave.http.SpanNameProvider;
+import com.github.kristofa.brave.jaxrs2.BraveTracingFeature;
+import java.io.IOException;
+import java.net.URI;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+import okhttp3.HttpUrl;
+import org.glassfish.grizzly.http.server.HttpServer;
+import org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpServerFactory;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.junit.After;
+import org.junit.AssumptionViolatedException;
+import org.junit.Test;
+
+public class ITBraveTracingFeature_Server extends ITHttpServer {
+
+  @Override
+  @Test
+  public void reportsSpanOnTransportException() throws Exception {
+    // TODO: it seems the only way to process exceptions in a standard way is ExceptionMapper.
```

Comment:
@lijunyong can you raise a separate issue on the lack of "error" annotation in jaxrs-2 container filters. I've jotted down the reason why it currently doesn't work here.